src/fides/common/api/v1/urn_registry.py

```diff
@@ -73,10 +73,14 @@
 PRIVACY_REQUEST_BULK_RETRY = "/privacy-request/bulk/retry"
 PRIVACY_REQUEST_DENY = "/privacy-request/administrate/deny"
 REQUEST_STATUS_LOGS = "/privacy-request/{privacy_request_id}/log"
+REQUEST_TASKS = "/privacy-request/{privacy_request_id}/tasks"
+PRIVACY_REQUEST_ACCESS_DATA = "/privacy-request/{privacy_request_id}/access_results"
+PRIVACY_REQUEST_TASK_QUEUE = (
+    "/privacy-request/{privacy_request_id}/tasks/{task_id}/queue"
+)
+
 PRIVACY_REQUEST_VERIFY_IDENTITY = "/privacy-request/{privacy_request_id}/verify"
 PRIVACY_REQUEST_RESUME = "/privacy-request/{privacy_request_id}/resume"
-PRIVACY_REQUEST_MANUAL_INPUT = "/privacy-request/{privacy_request_id}/manual_input"
-PRIVACY_REQUEST_MANUAL_ERASURE = "/privacy-request/{privacy_request_id}/erasure_confirm"
```

Comment:
Callback endpoints that supported the deprecated v1 manual connector that assumed sequential execution.